src/sync-accumulator.ts

```diff
@@ -78,6 +78,7 @@ export interface ITimeline {
 export interface IJoinedRoom {
     "summary": IRoomSummary;
     "state": IState;
+    "state_after"?: IState;
```

Comment:
Shouldn't this be unstable-prefixed in case the format changes by the time the MSC is merged and potentially corrupts the sync accumulator?

Reply:
Yes, it should be. The MSC has unstable prefixes and this one slipped through. I will update it.